src/style.css

```diff
@@ -6,7 +6,8 @@
 
 body {
   box-sizing: border-box;
-  overflow: hidden;
+
+  /* overflow: hidden; */
   margin: auto;
 }
```

Comment:
- [ ] Please do well to remove any spaces and comments in your code. This is to ensure you have a much cleaner code.

Reply:
Yes I have removed all the comments added